### DISCUSSION TITLE
Missing TypeScript definitions

### DISCUSSION THREAD

**Discussion by KosGrillis** [TanStack/table#4023] [General]
![image](https://user-images.githubusercontent.com/17753555/173729279-a7584876-9ade-4d2e-b87a-f9f89c38d3c8.png)

package.json specifies `"typings": "types/index.d.ts"` however that folder and file don't exist in the bundled packages. Looks to be located at `src/types.ts` instead

**Comment by masawik** [comment]
use
`npm install @tanstack/react-table@beta`
